CodeEdit/Features/WindowCommands/ViewCommands.swift

```diff
@@ -45,26 +45,35 @@ struct ViewCommands: Commands {
             }
             .keyboardShortcut("p", modifiers: [.shift, .command])
 
-            Button("Increase font size") {
-                if !(editorFontSize >= 288) {
-                    editorFontSize += 1
-                }
-                if !(terminalFontSize >= 288) {
-                    terminalFontSize += 1
+            Menu("Font Size") {
+                Button("Increase") {
+                    if !(editorFontSize >= 288) {
+                        editorFontSize += 1
+                    }
+                    if !(terminalFontSize >= 288) {
+                        terminalFontSize += 1
+                    }
                 }
-            }
-            .keyboardShortcut("+")
-            .disabled(windowController == nil)
+                .keyboardShortcut("+")
 
-            Button("Decrease font size") {
-                if !(editorFontSize <= 1) {
-                    editorFontSize -= 1
+                Button("Decrease") {
+                    if !(editorFontSize <= 1) {
+                        editorFontSize -= 1
+                    }
+                    if !(terminalFontSize <= 1) {
+                        terminalFontSize -= 1
+                    }
                 }
-                if !(terminalFontSize <= 1) {
-                    terminalFontSize -= 1
+                .keyboardShortcut("-")
+
+                Divider()
+
+                Button("Reset") {
+                    editorFontSize = 12
+                    terminalFontSize = 12
                 }
```

Comment:
Maybe fine for now, but this shouldn't be hardcoded. If someone has bad vision, that person should be able to set a standard font size, or we should try to get the system font size (if possible)

Reply:
Shouldn't this be the value of `Font Size` from `Settings -> Text Editing -> Font Size` in CodeEdit? (Similar for Terminal if resetting both). Then you would set your preferred default font size and Reset would always revert to that.

Reply:
Yeah, I am now thinking that ⌘+ and ⌘- should increment and decrement a zoom factor that is them applied to the font size. But that may be confusing to users when they see a font size of 12 in settings and the text is huge in their editor. They may wrongly decrease the font size to compensate not realizing they are zoomed in. Maybe we show a notice that they are zoomed in or out if applicable with the option to reset the zoom by font size in settings. Thoughts?

Reply:
The way Xcode works is that `+` and `-` do actually increase the font size and the font settings update as you do it -- so you would never see the wrong number / size in the settings. Then when you do a reset, it restores it to the last manually entered value from the settings window.

It remember this across restarts as well.

So they probably have an AppSetting value for "current font size" and "last manually selected font size". "current font size" is what you see, and using + and - will update "current font size". A reset will set "current font size" to "last manually selected" font size.

Reply:
That sounds like a good idea to me. The question is, is this out of scope for this PR?

@luah5 are you comfortable doing this in this PR?

Reply:
> That sounds like a good idea to me. The question is, is this out of scope for this PR?
> 
> @luah5 are you comfortable doing this in this PR?

Yeah, I can do that over the weekend.

Reply:
Let's do this in another PR